content/docs/user-guide/data-management/index.md

```diff
@@ -0,0 +1,168 @@
+# Data Management for Machine Learning
+
+<!--
+## Data Management for Machine Learning
+-->
+
+Where and how to store data and ML model files is one of the first decisions
+your team will face. But traditional back-up strategies do not fit the data
+science lifecycle. Large files end up scattered throughout several buckets.
+Overlapping dataset versions coexist, causing leakage and inefficient use of
+space. The project evolution is harder to track. What was the name of the best
+model? Is it safe to delete `huge_split.zip`? Can others reproduce my results?
+
+![Direct access storage](/img/direct_access_storage.png) _The S3 bucket on the
+right is shared (and bloated) by several people and projects. You need to know
+the exact location of the correct files, and use cloud-specific tools (e.g. AWS
+CLI) to access them directly._
+
+To maintain control and visibility over all your data and models, DVC stores
+large files and directories for you in a structured way. It tracks them by
+logging their locations and unique descriptions in **code-like metafiles**.
+
+![DVC-cached storage](/img/dvc_managed_storage.png)
```

Comment:
still too many details on cache. I don't think it's needed. Cache becomes a central part of it. It's fine to show it, but let's not complicate the picture with hashes etc ... we need to zoom out a bit, make ML Project part more compact, more focused on it's structure

Reply:
Shared latest diagrams on Slack. Should be easy to remove some parts/simplify.